create-proxy.sh

```diff
@@ -61,6 +72,7 @@ docker create \
   -v ${DIR}/zabbix/ssl/keys:/var/lib/zabbix/ssl/keys \
   -v ${DIR}/zabbix/ssl/ssl_ca:/var/lib/zabbix/ssl/ssl_ca \
   -p 10051:10051 \
+  --link zabbix-java-gateway:zabbix-java-gateway \
```

Comment:
Container links are legacy and should not be used: https://docs.docker.com/network/links/

I think specifying same link name as container is also redundant.